modules/ROOT/pages/cassandra/cassandra-use-cases.adoc

```diff
@@ -0,0 +1,231 @@
+= Use Cases - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Example: Create a Cassandra Keyspace using Studio
+
+Create a Cassandra keyspace which groups column families, then create a Cassandra table.
+
+=== Create a Keyspace
+
+. Create a new Mule Project in Anypoint Studio and fill in the Cassandra credentials in: `src/main/resources/mule-app.properties`.
++
+[source,text,linenums]
+----
+config.host=<HOST>
+config.port=<PORT>
+config.keyspace=<KEY_SPACE>
+config.username=<USERNAME>
+config.password=<PASSWORD>
+----
++
+. Drag an HTTP connector onto the canvas and leave the default values for Host and Port, then set the path to: `/test/createKeyspace`.
++
+. Drag a Transform Message component from the Mule palette to the right (process) side of the flow, select the component, and set the output payload. For example:
++
+[source,dataweave,linenums]
+----
+%dw 2.0
+output application/java
+---
+{
+  "keyspaceName": payload.keyspaceName,
+  "replicationFactor": payload.replicationFactor,
+  "replicationStrategyClass": payload.replicationStrategyClass
+} as Object {
+  class : "org.mule.modules.cassandradb.api.CreateKeyspaceInput"
+}
+----
++
+. Drag a Cassandra component for Create Keyspace onto the canvas.
++
+[%header%autowidth.spread]
+|===
+|Field |Description
+|Host | Enter a host name, or IP address of a Cassandra node.
+|Port | Enter a port number. The default port is 9042.
+|Keyspace | Enter the Cassandra keyspace. A keyspace groups column families.
+|Username | Enter a Cassandra user name if you enabled PasswordAuthenticator in the Cassandra YAML file. If AllowAllAuthenticator is enabled, leave this value blank.
+|Password | Enter the password if you enabled PasswordAuthenticator. Otherwise leave this value blank.
+|===
++
+. Click the Test Connection option to confirm that Mule can connect with the Cassandra instance. If the connection is successful, click OK to save the configuration. Otherwise, review and correct any invalid parameters and retest.
```

Comment:
Should OK be bolded?

Reply:
Don't know, what does the Salesforce style guide say?

Reply:
yes, all UI controls are bold.

Reply:
Fixed.